app/ui.js

```diff
@@ -180,7 +180,6 @@ var UI;
             UI.initSetting('view_only', false);
             UI.initSetting('path', 'websockify');
             UI.initSetting('repeaterID', '');
-            UI.initSetting('token', '');
```

Comment:
Hmm... does this prevent token from working as a URL param?  If so, it should still be in there.  If we somehow broke passing a token as part of the URL at some point, then we need to fix that.